style-guide/fields.md

```diff
@@ -0,0 +1,7 @@
+# Field Value Subsection
+
+The Field Value subsection describes the value of a field. In DxEditor, use the Return Value section for the field value description.
```

Comment:
I thought I'd have a chance to explain the task. But I also wanted to remove any references to DxEditor and DxStudio as well.